Library/Homebrew/formula_installer.rb

```diff
@@ -201,12 +202,16 @@ def check_install_sanity
   end
 
   def build_bottle_preinstall
-    @etc_var_glob ||= "#{HOMEBREW_PREFIX}/{etc,var}/**/*"
-    @etc_var_preinstall = Dir[@etc_var_glob]
+    @etc_var_dirs ||= ["#{HOMEBREW_PREFIX}/etc", "#{HOMEBREW_PREFIX}/var"]
+    @etc_var_preinstall = @etc_var_dirs.select { |d| File.directory?(d) }
```

Comment:
```suggestion
    @etc_var_preinstall = Find.find(@etc_var_dirs).to_a
```

not tested but from the API docs this looks like it should do what you want? Might need a `flatten` or similar but `Find.find` does accept multiple arguments and would be good to verify that it does blow up on e.g. missing directories so that check is needed.

Reply:
> `Find.find` does accept multiple arguments

That's useful to know - thanks!

> would be good to verify that it does blow up on e.g. missing directories so that check is needed.

It does. That was the initial test failure I had to deal with. It throws `Errno::ENOENT` when you do a `to_a`.